rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1132,8 +1133,7 @@ private void processPegoutsInBatch(
                 }
 
                 BtcTransaction generatedTransaction = result.get().getBtcTx();
-                // TODO: Update to call addPegoutTxToReleaseTransactionSet with the RskHash that calls the updateCollections
-                releaseTransactionSet.add(generatedTransaction, rskExecutionBlock.getNumber());
+                addPegoutTxToReleaseTransactionSet(generatedTransaction, releaseTransactionSet, rskTx.getHash(), rskTx.getValue().toBitcoin());
```

Comment:
```suggestion
                addPegoutTxToReleaseTransactionSet(generatedTransaction, releaseTransactionSet, rskTx.getHash(), totalPegoutValue);
```
The rsk tx that calls updateCollections probably has value 0. We care about logging the amount requested to release

Reply:
Resolved